features.go

```diff
@@ -6,7 +6,8 @@ const (
 	TGCheckTxHashFeature = "tg:check-txhash"
 	// Enables hot wallet (users can submit Ethereum deposit tx hashes).
 	TGHotWalletFeature = "tg:hot-wallet"
-
+	//Check Zero amount withdrawal
```

Comment:
-> `// Enables prevention of zero amount token withdrawals in the Gateway contract`